kolibri/plugins/utils/__init__.py

```diff
@@ -375,6 +372,8 @@ def run_plugin_updates():
                     ", ".join(update_manager.errors)
                 )
             )
+        return True
+    return False
```

Comment:
minor stylistic note on code that pre-dated this PR: I usually like to bail early out of functions so the bulk of the logic isn't nested in a conditional:

```python
def run_plugin_updates():
    if not config["UPDATED_PLUGINS"]:
        return False

    logger.info(
        "Detected updates to plugins: {}".format(
            ", ".join(config["UPDATED_PLUGINS"])
        )
    )
    update_manager = PluginUpdateManager(config["UPDATED_PLUGINS"])
    update_manager.update_plugins()
    if update_manager.errors:
        raise PluginUpdateException(
            "{} failed to update, please restart Kolibri".format(
                ", ".join(update_manager.errors)
            )
        )
    return True
```

The same strategy can be used to flatten the function below:

```python
    if skip_update:
        return

    # Run any plugin specific updates here in case they were missed by
    # our Kolibri version based update logic.
    updated = run_plugin_updates() or updated

    if not updated:
        return

    logger.info("Copying updated static files")
    # Collect any static files
    collectstatic_options = {
        "interactive": False,
        "verbosity": 0,
        "clear": True,
    }
    if CAN_USE_SYMLINKS:
        collectstatic_options["link"] = True
    call_command("collectstatic", **collectstatic_options)
```

Reply:
I tried to do a minimal diff here to prevent merge conflicts with the changes that have happened in develop. Happy to make further updates once this gets there.